Client/src/app/components/auth/registration/registration.component.html

```diff
@@ -91,7 +94,7 @@ <h1 class="offset-md-6 offset-3">Register</h1>
       Register
     </button>
     <span class="col-md-3">
-      Have already an account?&nbsp;<a href="/login" class="underlineLink">Go to login</a>      
+      Have already an account?&nbsp;<a href="/login" class="underlineLink">Go to login</a>
```

Comment:
This should read: Already have an acount?

Reply:
> And i just saw the link back to the jokes does not work anymore.

What link? /jokes?

Reply:
Sorry I should have been more clear. The left  button on the navbar does not link back to jokes.

Reply:
> 
> 
> Sorry I should have been more clear. The left button on the navbar does not link back to jokes.

No problem. Done.